airbyte-ci/connectors/pipelines/pipelines/airbyte_ci/connectors/migrate_to_base_image/commands.py

```diff
@@ -79,9 +79,6 @@ async def migrate_to_base_image(
         ctx.obj["concurrency"],
         ctx.obj["dagger_logs_path"],
         ctx.obj["execute_timeout"],
-        pull_request_number,
-        changelog,
-        bump,
```

Comment:
Going unix style: the bump and pull request pr can be used after this command if needed.